ui/apps/platform/src/constants/workflowPages.constants.ts

```diff
@@ -0,0 +1,31 @@
+export const WIDGET_PAGINATION_START_OFFSET = 0;
+
+export const OVERVIEW_LIMIT = 5;
+
+export const DASHBOARD_LIMIT = 8;
+
+export const LIST_PAGE_SIZE = 25;
+
+export const defaultCountKeyMap = {
+    CLUSTER: 'clusterCount',
+
+    COMPONENT: 'componentCount',
+    NODE_COMPONENT: 'nodeComponentCount',
+    IMAGE_COMPONENT: 'imageComponentCount',
+
+    CVE: 'vulnCount',
+    CLUSTER_CVE: 'clusterVulnerabilityCount',
+    IMAGE_CVE: 'imageVulnerabilityCount',
+    K8S_CVE: 'vulnCount: k8sVulnCount', // was broken, is it used?
```

Comment:
🤔 hard to say due to the dynamic nature of the sections this is used and the generic way the pages are composed. Leaving it as-is seems safest though unless someone has the time and desire to dig in more.